lib/cocoapods/sources_manager.rb

```diff
@@ -88,13 +88,18 @@ def update(source_name = nil, show_output = false)
         end
 
         changed_spec_paths = {}
+        # Ceate the Spec_Lock file if needed and lock it so that concurrent
+        # repo updates do not cause each other to fail
+        f = File.open("#{Config.instance.repos_dir}/Spec_Lock", File::CREAT)
```

Comment:
can we use the block form of `File.open`, since this will leak file handles if an exception occurs?